src/components/AvatarGroup/AvatarGroup.jsx

```diff
@@ -7,6 +7,10 @@ import Tooltip from "../Tooltip/Tooltip";
 import AvatarGroupCounter from "./AvatarGroupCounter";
 import styles from "./AvatarGroup.module.scss";
 
+const avatarOnClick = avatarProps => {
```

Comment:
why do we need a generator function?

Reply:
Cause in Avatar according whether onClick event was passed or not ClickableWrapper is applying. And if I'd leave it as it were `onClick={() => avatarOnClick(avatar.props)}` it would have always been "clickable" as there always some function